### PR TITLE
Add python code for Prefix_Sums.mdx, Greedy.mdx, and Problem Subordinates

### PR DESCRIPTION
**Pull request by topazand**

### CODE REVIEW

File: content/3_Silver/Prefix_Sums.mdx [hunk 1]
```diff
@@ -395,6 +395,21 @@ int main() {
 
 </CPPSection>
 
+<PySection>
+
+```python
+def psum(a):
+	s=0
```
Comment (HarryWangATX) [inline content/3_Silver/Prefix_Sums.mdx:402]: I'm not very familiar with python but I think you can get rid of the s variable and just use the psum array

**Comment by HarryWangATX** [comment]
Not too good at python, sorry. @frodakcin can you check this?

File: content/3_Silver/Prefix_Sums.mdx (hunk 1 above)
Comment (SansPapyrus683) [reply content/3_Silver/Prefix_Sums.mdx:402]: Yeah- it's possible for you to just set the first element as 0 and then use the last element of the array as the `s` variable.

Comment (HarryWangATX) [reply content/3_Silver/Prefix_Sums.mdx:402]: Okay cool. Can you modify your program and get rid of s pls. thx!

Comment (SansPapyrus683) [reply content/3_Silver/Prefix_Sums.mdx:402]: (if you didn't notice i'm not the author lol, just was giving my 2 cents)

Comment (topazand) [reply content/3_Silver/Prefix_Sums.mdx:402]: ok I can do this. But why do we want to get rid of this variable? I think this makes the code clearer. Or you want the code to be consistent with the C++ version?

Comment (SansPapyrus683) [reply content/3_Silver/Prefix_Sums.mdx]: I don't think that it makes the code clearer. And when you're using names like `s`, might as well just make it concise as well.

File: content/3_Silver/Greedy.mdx [hunk 2]
```diff
@@ -198,6 +198,22 @@ pw.close();
...
 </JavaSection>
 
+<PySection>
+
+```python
+# read in the input, store the algorithms in a list, algorithms
+algorithms.sort();
+count = 0 # number of minutes used so far
+i = 0
+while count + algorithms[i] <= x:
```
Comment (frodakcin) [inline content/3_Silver/Greedy.mdx:208]: array overflow? Is this code tested?

File: content/3_Silver/Intro_Tree.mdx [hunk 3]
```diff
@@ -275,6 +275,33 @@ public class Subordinates
 
 </JavaSection>
 
+<PySection>
+
+In the Python solution, we need to set recursion limit to $2\times10^5$.
```
Comment (frodakcin) [inline content/3_Silver/Intro_Tree.mdx:280]: Add comments explaining the code (check cpp/java ex)

File: content/3_Silver/Prefix_Sums.mdx [hunk 4]
```diff
@@ -395,6 +395,19 @@ int main() {
 
 </CPPSection>
 
+<PySection>
+
+```python
+def psum(a):
+	psum=[0]
+	for i in a:
+		psum.append(psum[-1]+i)
```
Comment (frodakcin) [inline content/3_Silver/Prefix_Sums.mdx:404]: add comment saying psum[-1] is last element?
or honestly, i think it's better to just write psum[i+1]=psum[i]+a[i]

also typo, should be a[i], not i

**Review by frodakcin — CHANGES_REQUESTED** [review]
Please thoroughly test code

File: content/3_Silver/Prefix_Sums.mdx (hunk 4 above)
Comment (frodakcin) [reply content/3_Silver/Prefix_Sums.mdx:404]: JK just noticed it's `for i in a`, myb

File: content/3_Silver/Greedy.mdx (hunk 2 above)
Comment (SansPapyrus683) [reply content/3_Silver/Greedy.mdx:208]: IIRC integer overflow isn't a problem for Python.

Comment (frodakcin) [reply content/3_Silver/Greedy.mdx:208]: array index out of bounds?

Comment (dongliu0426) [reply content/3_Silver/Greedy.mdx:208]: cpp and java code also do not check if `i` is out of bounds

Comment (topazand) [reply content/3_Silver/Greedy.mdx:208]: thanks, I'll fix that. Please help me to check if the Java code is correct

File: content/3_Silver/Prefix_Sums.mdx (hunk 4 above)
Comment (topazand) [reply content/3_Silver/Prefix_Sums.mdx:404]: PYTHON

File: content/3_Silver/Greedy.mdx (hunk 2 above)
Comment (frodakcin) [reply content/3_Silver/Greedy.mdx]: Wish there was an actual judge to test this on. I definitely am not able to catch all bugs by sight.

**Review by frodakcin — APPROVED** [review]
Looks good to me. I'll wait for a couple other reviews before merging

**Review by dongliu0426 — APPROVED** [review]

**Pull request merged**